sass/chosen.scss

```diff
@@ -1,6 +1,35 @@
 $chosen-sprite: url('chosen-sprite.png') !default;
 $chosen-sprite-retina: url('chosen-sprite@2x.png') !default;
 
+$chosen-bg-color: #fff;
+$chosen-box-shadow-color: #000;
+$chosen-border-color: #aaa;
+$chosen-group-name-color: #999;
+$chosen-highlighted-color: #3875d7;
+$chosen-highlighted-second-color: #2a62bc;
+
+$chosen-search-choice-bg-color: #eee;
+$chosen-search-choice-disabled-bg-color: #e4e4e4;
+$chosen-search-choice-color: #333;
+$chosen-search-choice-bg: linear-gradient(#f4f4f4 20%, #f0f0f0 50%, #e8e8e8 52%, #eee 100%);
+$chosen-search-choice-disabled-color: #666;
+$chosen-search-choice-focus-bg-color: $chosen-search-choice-disabled-color;
+$chosen-search-choice-disabled-border-color: #ccc;
+$chosen-no-results-bg: #f4f4f4;
+
+$chosen-color: #222;
+$chosen-border-color: #5897fb;
```

Comment:
This is the first actual issue I see in the code. `$chosen-border-color` is already being defined above. Looking at the CSS, `#5897fb` is only used when active/focused. So I think this should be `$chosen-active-border-color`. I see some other active/focused-specific variables, and I think those variables should be grouped together, and have consistent naming.